src/sds.c

```diff
@@ -58,36 +58,42 @@ static inline int sdsHdrSize(char type) {
     return 0;
 }
 
+/* Returns the maximum len and alloc for a given type.
+ *
+ * The buffer includes header and null terminator. The total buffer size must be
+ * up to 2^5, 2^8, 2^16, and 2^32 bytes. We assume jemalloc uses these sizes for
+ * allocation. Otherwise jemalloc allocates larger buffer, which length can't be
+ * stored in alloc field. */
+static inline size_t sdsTypeMaxSize(char type) {
```

Comment:
Smallest of comments, can we move this function back to where it was so we keep the git history.

Reply:
We now use it in `sdsReqType`. Do you prefer to add a forward declaration of `sdsTypeMaxSize` to keep the git history?

Reply:
yes